src/Webfactory/Slimdump/Database/Dumper.php

```diff
@@ -60,16 +66,8 @@ public function enableForeignKeys()
     public function dumpSchema($table, Connection $db, $keepAutoIncrement = true, bool $noProgress = false)
     {
         $this->keepalive($db);
-        $this->output->writeln("-- BEGIN STRUCTURE $table", OutputInterface::OUTPUT_RAW);
-        $this->output->writeln("DROP TABLE IF EXISTS `$table`;", OutputInterface::OUTPUT_RAW);
-
-        $tableCreationCommand = $db->fetchColumn("SHOW CREATE TABLE `$table`", [], 1);
-
-        if (!$keepAutoIncrement) {
-            $tableCreationCommand = preg_replace('/ AUTO_INCREMENT=\d*/', '', $tableCreationCommand);
-        }
```

Comment:
I don't like that now the `SqlDumper` is concerned with querying schema information from the database. If possible, I'd like to avoid giving it that responsibility.

On the other hand, we need the exact SQL statement for the export and we need it only for the SQL export case (no idea how we could use it for CSV in a meaningful way). 

So, for now, let's try it that way and see where it leads us.